CHANGELOG.md

```diff
@@ -1,8 +1,14 @@
 
 
-## [Unreleased](https://git.openjdk.org/jtreg/compare/jtreg-7.1+1...master)
+## [Unreleased](https://git.openjdk.org/jtreg/compare/7.1.1+1...master)
```

Comment:
You dropped the `jtreg-` ... that seems wrong...

Reply:
Fixed.